src/app/offers/single-offer-item/single-offer-item.component.scss

```diff
@@ -8,13 +8,16 @@
   background: $background-color;
   padding-bottom: 10px;
   transition: all 0.3s cubic-bezier(.25, .8, .25, 1);
-  cursor: pointer;
 }
 
 .card-right .app-basic-title {
   display: none;
 }
 
+.app-basic-title {
+  cursor: pointer;
+}
+
```

Comment:
if a title must be clickable, wrap it in an anchor tag, don't use `cursor: pointer`.